db/migrate/20191217135324_create_invoices.rb

```diff
@@ -0,0 +1,15 @@
+class CreateInvoices < ActiveRecord::Migration[6.0]
+  def change
+    create_table :invoices do |t|
+      t.date :date
+      t.date :due_date
+      t.float :subtotal
+      t.float :tax
+      t.float :total
+      t.boolean :paid_for
```

Comment:
A little too many attributes to start with. Feels like premature optimization.

Reply:
Did a proper ERB